test-resources/headless-core-stub/lambdas/start/src/start-handler.ts

```diff
@@ -1,11 +1,42 @@
 import type { LambdaInterface } from "@aws-lambda-powertools/commons/types";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { JWK, JWTPayload, KeyLike } from "jose";
+import { handleErrorResponse } from "./errors/error-response";
+import { generateJwtClaimsSet, parseJwtClaimsSetOverrides, validateClaimsSet } from "./services/jwt-claims-set-service";
+import { encryptSignedJwt, getPrivateSigningKey, getPublicEncryptionKey, signJwt } from "./services/signing-service";
+import { ClaimsSetOverrides } from "./types/claims-set-overrides";
+import { JWTClaimsSet } from "./types/jwt-claims-set";
+
+const logger = new Logger();
```

Comment:
Could we add [logger context](https://docs.powertools.aws.dev/lambda/typescript/1.4.1/core/logger/#inject-lambda-context) just in case please